templates/core/.env.sample

```diff
@@ -64,3 +64,6 @@ ENABLE_AIRLOCK_MALWARE_SCANNING=false
 
 # TODO: move to RP default with https://github.com/microsoft/AzureTRE/pull/2634
 WORKSPACE_APP_SERVICE_PLAN_SKU="P1v2"
+
+# The number of processes to start in the resource processor VMSS image
+RESOURCE_PROCESSOR_NUMBER_PROCESSES_PER_INSTANCE=5
```

Comment:
If we recommend 5 here, why do we have a default of 2?

Reply:
fair - have changed default to 5 as well